cmd/tealdbg/localLedger.go

```diff
@@ -211,6 +261,34 @@ func getAppCreatorFromIndexer(indexerURL string, indexerToken string, app basics
 	return creator, nil
 }
 
+func getAssetCreatorFromIndexer(indexerURL string, indexerToken string, asset basics.AssetIndex) (basics.Address, error) {
+	queryString := fmt.Sprintf("%s/v2/assets/%d", indexerURL, asset)
+	client := &http.Client{}
+	request, err := http.NewRequest("GET", queryString, nil)
+	request.Header.Set("X-Indexer-API-Token", indexerToken)
+	resp, err := client.Do(request)
+	if err != nil {
+		return basics.Address{}, fmt.Errorf("asset request error: %s", err)
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != 200 {
```

Comment:
nit: is there any constant in the http package that equal to the magic number 200 ? Maybe StatusOK ?